esp/services/ws_packageprocess/ws_packageprocessService.cpp

```diff
@@ -735,6 +812,87 @@ bool CWsPackageProcessEx::onAddPackage(IEspContext &context, IEspAddPackageReque
     return true;
 }
 
+#define PACKAGEMAP_PATH_SEP_CHAR '/'
+
+bool nextPmPathNode(const char *&path, StringBuffer &node)
+{
+    if (*path==PACKAGEMAP_PATH_SEP_CHAR)
```

Comment:
This is an internal highly repeated function call.. it would be unnecessary overhead, it's enough to check that path isn't null once at the beginning.